api/v1alpha1/hostedcluster_types.go

```diff
@@ -538,6 +547,111 @@ type IBMCloudPlatformSpec struct {
 	ProviderType configv1.IBMCloudProviderType `json:"providerType,omitempty"`
 }
 
+// PowerVSPlatformSpec defines IBMCloud PowerVS specific settings for components
+type PowerVSPlatformSpec struct {
+	// ResourceGroup is the IBMCloud Resource Group in which the cluster resides. If not
+	// specified then default resource group of the account will be used.
+	//
+	// +immutable
+	ResourceGroup string `json:"resourceGroup,omitempty"`
+
+	// Region is the IBMCloud region in which the cluster resides. This configures the
+	// OCP control plane cloud integrations, and is used by NodePool to resolve
+	// the correct boot image for a given release.
+	//
+	// +immutable
+	Region string `json:"region"`
+
+	// Zone is the availability zone where control plane cloud resources are
+	// created.
+	//
+	// +immutable
+	Zone string `json:"zone,omitempty"`
+
+	// Subnet is the subnet to use for control plane cloud resources.
+	//
+	// +optional
+	Subnet *PowerVSResourceReference `json:"subnet,omitempty"`
+
+	// ServiceInstanceID is the ServiceInstance to use for control plane cloud resources.
+	ServiceInstanceID string `json:"serviceInstanceID,omitempty"`
+
+	// VPC specifies IBM Cloud PowerVS Load Balancing configuration for the control
+	// plane.
+	//
+	// +immutable
+	VPC *PowerVSVPC `json:"vpc,omitempty"`
+
+	// KubeCloudControllerCreds is a reference to a secret containing cloud
+	// credentials with permissions matching the cloud controller policy. The
+	// secret should have exactly one key, `credentials`, whose value is an AWS
+	// credentials file.
+	//
+	// TODO(dan): document the "cloud controller policy"
+	//
+	// +immutable
+	KubeCloudControllerCreds corev1.LocalObjectReference `json:"kubeCloudControllerCreds"`
```

Comment:
I brought this up in the past: I wonder if both KubeCloudControllerCreds and NodePoolManagementCreds should belong to the generic spec.